04-files.md

```diff
@@ -21,14 +21,14 @@ that finds files whose names match a pattern.
 We provide those patterns as strings:
 the character `*` matches zero or more characters,
 while `?` matches any one character.
-We can use this to get the names of all the html files:
+We can use this to get the names of all the csv files in the data directory:
```

Comment:
The code below does not use the "data" directory, so maybe this should say "current" directory, since that's what it does.